Automattic/wp-calypso
# Checkout: Update UI of Pix payment method after feedback

## Proposed Changes

This PR makes some small UI updates to the Pix payment method UI after testing feedback in pdtkmj-2nb-p2

Specifically, the term "CPF" is used for the CPF number which will be more familiar to BRL users, and we changed the QR code page to use the term "Pay with Pix" instead of "Confirm your payment" to be clear that the paying actually needs to happen. We also changed the instructions on the QR code page to read "using your banking app" instead of "using the Pix app" because there is no Pix app.

This PR also replaces the "On mobile?" section to include instructions and a button to copy the code required to activate Pix manually. The previous "redirect" text was less useful because it would just bring you to an Ebanx page with the same code.

Before             |  After
:-------------------------:|:-------------------------:
<img width="593" alt="Screenshot 2024-02-06 at 6 12 19 PM" src="https://github.com/Automattic/wp-calypso/assets/2036909/f8a2f081-d0a2-4aee-a655-d08bd8e973d6"> | <img width="579" alt="Screenshot 2024-02-06 at 6 13 02 PM" src="https://github.com/Automattic/wp-calypso/assets/2036909/b21d714f-e453-44aa-b354-03bf92859ae9">
<img width="540" alt="Screenshot 2024-02-06 at 6 12 30 PM" src="https://github.com/Automattic/wp-calypso/assets/2036909/a9bbef17-2fc3-4e05-b8e9-7a9793cc5cf2"> | <img width="463" alt="Screenshot 2024-02-13 at 2 20 49 PM" src="https://github.com/Automattic/wp-calypso/assets/2036909/49a634fc-52ed-491b-889b-aea8a079b316">

Fixes https://github.com/Automattic/payments-shilling/issues/2411
Fixes https://github.com/Automattic/payments-shilling/issues/2429

## Testing Instructions

- To use Pix as a payment method option in checkout, first you'll need to have BRL as your currency, and then you'll need to select "Brazil" as the country inside checkout (and a Brazilian postal code like `61919-230`).
- Use Ebanx [test customer info](https://developer.ebanx.com/docs/resources/testCustomerData/) to fill out the Pix form. Verify that the "CPF" field reads "CPF" instead of "Taxpayer Identification Number".
- Click to submit the payment.
- Verify that the QR code screen reads `Pay with Pix` and that the instructions read `Please scan the QR code using your banking app to complete your XXX payment`. Make sure that the instructions all look good and make sense.
- Click the "Copy" button and make sure that it copies the correct code into your clipboard.
